src/js/containers/SignUpPage/SignUpPage.js

```diff
@@ -6,109 +6,134 @@ import _ from 'lodash';
-                                </div>
-                            </div>
-                            <div className="control-group">
-                                <div className="controls">
-                                    <label className="checkbox"><input ref="termsOfService" required="required" type="checkbox"/>
-                                        <div className="controls">
-                                            <p><FormattedMessage id="container.signup.form.termsAndConditions.iAgreeTo"/> <a href={TERMS_AND_CONDITIONS_PAGE} target="_blank"><FormattedMessage id="container.signup.form.termsAndConditions.cloudFlaresTermsAndConditions"/></a> <FormattedMessage id="container.signup.form.termsAndConditions.and"/> <a href={PRIVACY_POLICY_PAGE} target="_blank"><FormattedMessage id="container.signup.form.termsAndConditions.privacyPolicy"/></a><FormattedMessage id="container.signup.form.termsAndConditions.period"/></p>
-                                        </div>
-                                    </label>
-                                    </div>
-                                </div>
-                            <div className="control-group">
-                                <div className="controls">
-                                    <button type="submit" className="btn btn-success btn-large width-full">
-                                        <FormattedMessage id="container.signup.form.button" />
-                                    </button>
-                                </div>
-                            </div>
-                        </fieldset>
-                    </form>
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      password: '',
+      passwordConfirm: '',
+      termsOfService: false
+    };
+  }
+
+  render() {
+    let { formatMessage } = this.props.intl;
+
+    let overflowStyle = { overflow: "hidden", marginBottom: '2.5rem' };
+
+    return (
+        <div id="cf-login-page" style={{ margin: "2rem auto", maxWidth: '400px' }}>
+          <Form layout="vertical" onSubmit={(e) => this.handleSignUpSubmit(e)}>
+            <LayoutContainer>
+              <div style={overflowStyle}>
+                <LayoutRow>
+                  <LayoutColumn width={1/1}>
+                    <FormHeader title={formatMessage({ id: 'container.signup.form.title' })}/>
+                  </LayoutColumn>
+                </LayoutRow>
+              </div>
+              <FormFieldset legend="">
+                <div style={overflowStyle}>
+                  <LayoutRow>
+                    <LayoutColumn width={1/1}>
+                      <FormLabel hidden><FormattedMessage id="container.signup.form.email"/></FormLabel>
+                      <Input name="email" type="text" value={this.state.email} onChange={email => this.setState({ email: email })} placeholder={formatMessage({ id: 'container.signup.form.email' })}/>
```

Comment:
https://github.com/cloudflare/cloudflare-plugin-frontend/blob/master/.babelrc#L2

I thought `2015` was compiling only up to ES2015 to commonjs. Looks like it compiles ES6 as well. https://www.npmjs.com/package/babel-preset-es2015